lib/datadog/tracing/contrib/configurable.rb

```diff
@@ -41,7 +41,7 @@ def configurations
 
           # Create or update configuration associated with `matcher` with
           # the provided `options` and `&block`.
-          def configure(matcher = :default, options = {}, &block)
+          def configure(matcher = :default, options = {}, tracing = nil, &block)
```

Comment:
Should this be `tracing_configuration` or `tracing_instrumentation` instead maybe?

Reply:
Yeah it's a bit ambiguous. Since this is a `Configuration` object and the other name here is `config` I went with `tracing_config`.